pkg/installmanager/installmanager.go

```diff
@@ -231,26 +232,16 @@ func (m *InstallManager) Run() error {
 
 	m.waitForInstallerBinaries()
 
-	// Generate an install-config.yaml:
-	sshKey := os.Getenv("SSH_PUB_KEY")
-	pullSecret := os.Getenv("PULL_SECRET")
-	m.log.Info("generating install config")
-	ic, err := install.GenerateInstallConfig(cd, sshKey, pullSecret, true)
+	m.log.Info("copying install-config.yaml")
+	destInstallConfigPath := filepath.Join(m.WorkDir, "install-config.yaml")
+	cmd := exec.Command("cp", m.InstallConfigMountPath, destInstallConfigPath)
```

Comment:
Should we copy the entire /installconfig directory over and not be proscriptive about what files are needed? I can think of a case where a future version of openshift-install uses different files.

Reply:
That's reasonable.